src/features/home/pages/Home.test.tsx

```diff
@@ -1,7 +1,9 @@
 import { render, fireEvent, waitFor } from '@testing-library/react-native'
 import React from 'react'
 
+import * as api from 'features/auth/api'
```

Comment:
Ok de renommer, mais il faut apporter du contexte alors.
Si on voulait enforcer cela, il faudrait déclarer nos fonctions sur un objet `authApi` dans `features/auth/api`.
```suggestion
import * as authApi from 'features/auth/api'
```

Reply:
Vu ensemble, c'est ok.